sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/metrics/_exporter.py

```diff
@@ -75,13 +76,15 @@ class AzureMonitorMetricExporter(BaseExporter, MetricExporter):
     """Azure Monitor Metric exporter for OpenTelemetry."""
 
     def __init__(self, **kwargs: Any) -> None:
+        self._is_sdkstats = kwargs.get("is_sdkstats", False)
```

Comment:
We move these attribute settings BEFORE the parent `__init__` calls because `BaseExporter` needs to know if these fields are set.